src/core/algorithms/statistics/data_stats.cpp

```diff
@@ -35,12 +35,12 @@ void DataStats::ResetState() {
 }
 
 Statistic DataStats::GetMin(size_t index, mo::CompareResult order) const {
-    mo::TypedColumnData const& col = col_data_[index];
+    mo::TypedColumnData const &col = col_data_[index];
```

Comment:
such fixes looks strange, seems that we need to return it to how it was (in all places)